data_pipeline/testing_helpers/containers.py

```diff
@@ -36,8 +36,11 @@
 
 class ContainerUnavailableError(Exception):
     def __init__(self, project='unknown', service='unknown'):
-        msg = open("logs/docker-compose.log", 'r').read()
-        Exception.__init__(self, "Container for project {0} and service {1} failed to start with msg {2}".format(
+        try:
+            msg = open("logs/docker-compose.log", 'r').read()
+        except IOError:
+            msg = ""
```

Comment:
Right now if it can't find the message it will only say:
"Container for project {0} and service {1} failed to start with {2}" if the log is not found, can you make the IOError message "no error log found"

That way it reads "Container for project {0} and service {1} failed to start with no error log found", which is a little more informative.